### DISCUSSION TITLE
Horizontal alignment, cursor position

### DISCUSSION THREAD

**Discussion by hamanuha** [dm4t2/vue-currency-input#377] [Ideas]
First, I want to thank you for this great component and the effort you put into this.

I only have a small feature wish / idea:
If I align the text on the right and I click on the input field, the cursor jumps to the first position.
It would be great to set an option, that on focus the cursor jumps to the last position.

**Comment by dm4t2** [comment]
Hi, thanks for your suggestion. I will look into it 👍

**Reply by hamanuha** [reply]
Thank you!
